inst/include/epiworld/epiworld.hpp

```diff
@@ -18,8 +18,8 @@
 
 /* Versioning */
 #define EPIWORLD_VERSION_MAJOR 0
-#define EPIWORLD_VERSION_MINOR 3
-#define EPIWORLD_VERSION_PATCH 2
+#define EPIWORLD_VERSION_MINOR 4
+#define EPIWORLD_VERSION_PATCH 3
```

Comment:
This needs to be reflected in the DESCRIPTION file of the R package. Can you ensure that is the case and also create an issue for a CI workflow that checks this is always the case?

Reply:
How do you want this reflected in the R package? Do you want the `Version: 0.3-2` to always match the C++ versions being used? Also, is there are reason we use `0.3-2` instead of `0.3.2`?

Reply:
The `-` in the version is a convention I've been using for a time now. There's no particular reason why I chose that. Nonetheless, we cannot change that now because that's what is on CRAN.

Reply:
Done